_docs_integrate/data-model-overview.md

```diff
@@ -571,11 +571,42 @@ Response Items are sent inside of a Response. They contain the response data tha
 
 #### AcceptResponseItem
```

Comment:
Mhh, I'd add a sentence, since the derivation of the AcceptResponseItem does not only depends on the actual RequestItem (as it is stated in the introductory text), but on the used AcceptRequestItemParameters as well. Perhaps just link to the given overview of RequestItems and their corresponding ResponseItems (https://enmeshed.eu/integrate/request-and-response-introduction#types-of-requestitems) in one sentence at the end of this paragraph. Overall, of course, it is not a must to add it here, but I think it would be helpful to a reader of this page.